persistence-modules/hibernate5/src/main/java/com/baeldung/hibernate/findall/FindAll.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.hibernate.findall;
+
+import java.util.List;
+
+import javax.persistence.TypedQuery;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+
+import org.hibernate.Session;
+
+import com.baeldung.hibernate.pojo.Student;
+
+public class FindAll {
+
+    private Session session;
+
+    public FindAll(Session session) {
+        super();
+        this.session = session;
+    }
+
+    public List<Student> findAllWithJpql() {
+        return session.createQuery("Select a From Student a", Student.class)
```

Comment:
Can you clean up the capitalization of this query? I believe the JPQL standard is to use the same conventions as SQL/MySQL, isn't it?

Reply:
JPQL doesn't have capitalization convention as far as i know. Some projects use uppercase, others lowercase etc. I guess we should define unique style for our tutorials. I uppercased both keywords ...